README.md

```diff
@@ -98,7 +98,7 @@ Note that you should configure accessing the SCIM bridge credentials through eit
 | file | string | `scimsession` | File name. |
 | accessModes | list | `[ReadWriteOnce]` | Access modes. |
 | resources | object | `{ "requests": { "storage": "1Gi" } }` | The default storage request is `1Gi`. |
-| storageClass | string | N/A | Storage class. The default is not set since `""` has a specific meaning. `do-block-storage` is recommended for Digital Ocean. |
+| storageClass | string | N/A | Storage class. The default is not set. Set to `"-"` to set value to `""`. `do-block-storage` is recommended for Digital Ocean. |
```

Comment:
```suggestion
| storageClass | string | unset/nil/not set | Storage class. Set to `"-"` to set value to `""` in resulting application. `do-block-storage` is recommended for Digital Ocean. |
```
Looking at the rendered output, I think we should state the default is not set in the default column. I also think explaining the `-` choice slightly would be helpful. 

![Screenshot 2021-12-21 at 2 38 16 PM](https://user-images.githubusercontent.com/7812612/146994634-ef37afa2-ffb0-4290-99cf-1d894e1cbf0f.png)